### ISSUE TITLE
main::load_credentials: no credentials were loaded

### ISSUE THREAD

**Issue by ibrokemypie** [mariusor/mpris-scrobbler#34]
Hey, whenever I try to use any of the authentication commands I get the following output:
```
WARNING main::loading_credentials: failed
WARNING main::load_credentials: no credentials were loaded
````
anything I am doing wrong?

**Comment by mariusor** [comment]
Please post the whole output for what you're trying and add which linux distribution you're using.

It might be a bug in how we create the credentials file.

**Comment by Viktini** [comment]
I have this bug as well and I'm running Arch Linux (Linux 4.15.12-1-ARCH x86_64).
Whenever I try to set the token for my ListenBrainz account, the error in OP's issue is shown and nothing else, even if I try to get more output through verbose parameters.

This also happens when attempting to add Last.fm or Libre.fm accounts.

**Comment by mariusor** [comment]
Please post full output of what you're trying, and **censor** the secret tokens.

Also post content of `~/.local/share/mpris-scrobbler/credentials` with same secret credentials censored if they are present, before and after running your command.

As a workaround for listenbrainz, you can edit manually the credentials file:

```

[listenbrainz]
enabled = true
token = e92c33**-****-****-****-********848a
```

**Comment by ibrokemypie** [comment]
```
mpris-scrobbler-signon token listenbrainz
WARNING main::loading_credentials: failed
WARNING main::load_credentials: no credentials were loaded
```
there is no ~/.local/share/mpris-scrobbler/credentials file.

This is on arch linux, and that is the full output

**Comment by mariusor** [comment]
OK. I see. 

I think the problem is due to an early exit either because there are no existing credentials, or because the mpris-scrobbler is running.

Either way I'm fixing it ASAP.

**Comment by mariusor** [comment]
Fixed in 15a3d2af476921. Tagged new release [v0.2.50](https://github.com/mariusor/mpris-scrobbler/releases/tag/v0.2.50). Released new version of the AUR package.
